language/tools/move-mv-llvm-compiler/src/stackless/translate.rs

```diff
@@ -496,6 +496,13 @@ impl<'mm, 'up> ModuleContext<'mm, 'up> {
             }
             Type::Primitive(PrimitiveType::Address) => self.get_llvm_type_for_address(),
             Type::Primitive(PrimitiveType::Signer) => self.get_llvm_type_for_signer(),
+
+            Type::Primitive(PrimitiveType::Num)
+            | Type::Primitive(PrimitiveType::Range)
+            | Type::Primitive(PrimitiveType::EventStore) => {
+                panic!("{mty:?} only appears in specifications.")
+            }
```

Comment:
Should we not just ignore these? That is what I did previously for the spec-related operations. As far as I can tell, there is nothing wrong with specs being in programs that we accept-- we just won't do any prover-like analysis on them. There are many programs in the various libraries (including move-stdlib that we don't yet build) that have specs along with the code, we don't want to crash or reject on those.